packages/manager/src/features/CloudPulse/shared/CloudPulseResourcesSelect.tsx

```diff
@@ -100,6 +119,26 @@ export const CloudPulseResourcesSelect = React.memo(
       // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [resources, region, xFilter, resourceType]);
 
+    // selected resources will appear at the top in the autcomplete popper
+    const sortedResourcesList = React.useMemo<CloudPulseResources[]>(() => {
+      return [...getResourcesList].sort((resource_a, resource_b) => {
```

Comment:
```suggestion
      return [...getResourcesList].sort((resourceA, resourceB) => {
```
or 

```suggestion
      return [...getResourcesList].sort((a, b) => {
```

Avoiding using snake cases for variables